src/index.js

```diff
@@ -23,6 +23,14 @@ module.exports = {
     node: true
   },
   extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  overrides: [
+    {
+      files: ['**/commands/**', '**/bin/**'],
```

Comment:
```suggestion
      files: ['**/bin/**', '**/scripts/**'],
```

I think we don't want to whitelist all the paths to `commands` (e.g., if someone is using DDD) or https://github.com/uphold/reward-service/tree/master/src/commands). I think `bin` and `scripts` are enough.

Reply:
Changed.